master/internal/db/postgres_experiments.go

```diff
@@ -396,7 +396,7 @@ FROM trials t
 WHERE t.id=$2
   AND s.state = 'COMPLETED'
   AND total_batches >= $3
-  AND total_batches <= $4
+  AND ($4 <= 0 OR total_batches <= $4)
```

Comment:
question: don't see a need for it, but it seems like just for reasons of consistency, `end_batches = 0` should be a valid filter. is there a reason we need this? 

is it since because we are passing `0` when no value is supplied?

Reply:
yes `end_batches = 0` should mean no filter on `end_batches` and make this line TRUE